src/sentry/taskworker/task.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import annotations
+
+from collections.abc import Callable
+from datetime import timedelta
+from functools import update_wrapper
+from typing import TYPE_CHECKING, Any
+
+from django.utils import timezone
+
+if TYPE_CHECKING:
+    from sentry_protos.sentry.v1.taskworker_pb2 import RetryState
```

Comment:
this seems wrong -- this doesn't look like an import cycle so this should just be a normal import